CMakeLists.txt

```diff
@@ -19,6 +19,9 @@ find_program(PYTHON REQUIRED NAMES python3 python)
 
 # Threads are required
 find_package(Threads REQUIRED)
+if (Threads_FOUND)
+  set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -pthread")
```

Comment:
This is not necessary. `find_package(THREADS)` sets compiler and linker flags with `target_link_libraries` (https://stackoverflow.com/a/49769972).